### DISCUSSION TITLE
Set deploy location and curlib via env var

### DISCUSSION THREAD

**Discussion by pabloto** [codefori/vscode-ibmi#1567] [General]
Hi, 

I'd like to have an env var or something else where I could to set the deploy location and the build library, so if I'm in a branch different from main/master branch , I could set a different deploy location and build library specific for the branch.

Is it possible to automate it? without remembering to set it manually?

Many thanks.

**Comment by worksofliam** [comment]
It might be possible, though we don't have a way to edit the path with variables.

@sebjulliand What do you think?

**Reply by sebjulliand** [reply]
That would be nice; however, I don't know if that would not be a good feature to add to the Project Explorer instead of the base.

**Reply by pabloto** [reply]
Hi, this is Edmund response:

https://github.com/IBM/vscode-ibmi-projectexplorer/discussions/215

Thank you

**Comment by pabloto** [comment]
Hi, 

do you think this implementation is possibile?

**Reply by worksofliam** [reply]
Hey @pabloto 

Yes, it is possible, but it will take some time to write the code to support it. I don't have a timeline for you, but it is possible for us to do.
